app/src/main/res/layout/activity_talk_reply.xml

```diff
@@ -91,18 +91,18 @@
                 android:singleLine="true"
                 android:ellipsize="end"/>
 
-            <ImageView
+            <TextView
                 android:id="@+id/replySaveButton"
-                android:layout_width="56dp"
-                android:layout_height="match_parent"
-                android:layout_gravity="end"
+                style="@style/TransparentButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="4dp"
                 android:background="?attr/selectableItemBackgroundBorderless"
```

Comment:
Since you are adding the `@style/TransparentButton`, the ripple animations will appear in both the foreground and background.